lib/module.js

```diff
@@ -74,6 +74,15 @@ function Module(id, parent) {
 }
 module.exports = Module;
 
+const builtinModules = Object.keys(process.binding('natives'))
+  .filter((module) => {
```

Comment:
Using a plain for-loop will be faster and we generally avoid using the functional array methods in node core.

Reply:
I think for code that’s run a single time, we can choose to prefer clarity over performance.